step.yml

```diff
@@ -68,6 +68,8 @@ inputs:
     opts:
       category: Debug
       summary: Reset repository contents with git reset --hard HEAD and git clean -f after repository updated
+      description: |-
+        Input values `Yes` and `No` are DEPRECATED, these value options will be removed in version 4.1.0
```

Comment:
Input values `Yes` and `No` are DEPRECATED, these value options will be removed in version 4.1.0. Please use `yes` and `no` instead.